distribution/docs/src/main/resources/_contents/_quickstart/quickstart-installing-contents.adoc

```diff
@@ -116,19 +116,15 @@ Prior to installing ${branding}, ensure the system time is accurate to prevent f
 
 === Quick Install of ${branding}
 
-.Check System Time
-[WARNING]
-====
-Prior to installing ${branding}, ensure the system time is accurate to prevent federation issues.
-====
-
 . Download the ${branding} {download-url}[zip file].
 . Install ${branding} by unzipping the zip file.
 +
 .Windows Zip Utility Warning
 [WARNING]
 ====
-The default Windows zip utility (such as double-clicking file) will not work to unzip the distribution, use Java or a third party utility instead.
+The Windows Zip implementation, which is invoked when a user double-clicks on a zip file in the Windows Explorer, creates a corrupted installation.
```

Comment:
not sure if it is possible but wondering if we can make a common comment so it can re-appear in multiple places or if this should be better resolved with a link. Seems like we would need to maintain this Windows warning in 2 places.